### PR TITLE
F ame additions

### PR DESCRIPTION
**Pull request by dg-shadow**
Adds functions needed for sr_manipulation improvements made for AME demo

### CODE REVIEW

**Comment by ugocupcic** [comment]
Should it be added to RobotCommander.rst ?

**Comment by dg-shadow** [comment]
Maybe... What is RobotCommander.rst? ;)

**Comment by ugocupcic** [comment]
https://github.com/shadow-robot/sr_interface/blob/F_ame_additions/sr_robot_commander/doc/tutorial/RobotCommander.rst

**Comment by dg-shadow** [comment]
Ok, up to you I guess.. Not all functions from robot commander are in there - my feeling is the ones I added are a bit low-level to be in there... Happy to add if you think it's worth it...

File: sr_robot_commander/doc/tutorial/RobotCommander.rst [hunk 1]
```diff
@@ -238,6 +238,93 @@ Example
...
+
+Argumeent is a list of waypoints, being dictionaries containing the name of the pose, the
+time taken to reach the it from the previous one, and optionally, the time to pause
+before the next.
+
+
+Example
+^^^^^^^
+
+.. code:: python
```
Comment (ugocupcic) [inline sr_robot_commander/doc/tutorial/RobotCommander.rst:280]: appears broken

**Pull request merged**